### ISSUE TITLE
Unable to start Open-mSupply due to missing Windows Library

### ISSUE THREAD

**Issue by jmbrunskill** [msupply-foundation/open-msupply#5023]
## What went wrong? 😲

![telegram-cloud-photo-size-1-4904719757610167657-y](https://github.com/user-attachments/assets/b8abb050-7208-4156-8d5b-43bc468515cd)

## Expected behaviour 🤔

Open-mSupply works!

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Install on windows 11?

## Your environment 🌱

- Open mSupply Version: 2.2.2
- OS: Windows 11 (without C Runtime)

**Comment by lache-melvin** [comment]
@jmbrunskill does this need a patch into 2.2 or is this thinking that we resolve for next release? I seem to have missed the context on this one 😅

**Comment by jmbrunskill** [comment]
I think it can be in develop but high priority, would need to check with Adam, but I think the active deployments are done? I'll double check...

**Comment by mark-prins** [comment]
yeah, we do have a workaround and the next release isn't far away now

**Comment by jmbrunskill** [comment]
https://stackoverflow.com/questions/31770604/how-to-generate-statically-linked-executables/44387312#44387312

**Comment by andreievg** [comment]
Closing until it re-appears

**Comment by mark-prins** [comment]
Reviving the issue as this has been happening often. latest is on windows 10

![Image](https://github.com/user-attachments/assets/23b6115e-580b-4ed8-a42b-3396e706a176)

can we implement the static linking please
